src/pudl_archiver/metadata/sources.py

```diff
@@ -398,4 +398,22 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "epapcap": {
+        "title": "EPA -- Priority Climate Action Plan",
```

Comment:
In the past we've put the abbreviated name before the double-dash and then expanding the name after it. Are the abbreviations intentionally being left out? They appear to be present in some of the new dataset titles, but missing in others.

```suggestion
        "title": "EPA PCAP -- Priority Climate Action Plan",
```